pkg/notes/model/note.ts

```diff
@@ -0,0 +1,102 @@
+import type { AccountID } from '../../accounts/model/account.js';
+import type { ID } from '../../id/type.js';
+
+export type NoteID = string;
+export type NoteVisibility = 'public' | 'home' | 'followers' | 'direct';
```

Comment:
```suggestion
export type NoteVisibility = 'PUBLIC' | 'HOME' | 'FOLLOWERS' | 'DIRECT';
```
まだ取り決めしていなかったことだと思いますが, ニーモニックコード (意味が直接読み取れる符号) は SCREAMING_SNAKE_CASE にしておいたほうがデバッグ時に出現した場合などに識別したり取り扱ったりしやすいと思います.